test/server/graphql/v1/createOrder.incognito.test.js

```diff
@@ -111,12 +111,18 @@ describe('server/graphql/v1/createOrder.incognito', () => {
       expect(collectiveData.orders[0].fromCollective.createdByUser.lastName).to.be.null;
       expect(collectiveData.orders[0].fromCollective.createdByUser.email).to.be.null;
       expect(collectiveData.orders[0].fromCollective.createdByUser.firstName).to.be.null;
-      expect(collectiveData.members[0].member.createdByUser.firstName).to.equal('admin');
-      expect(collectiveData.members[1].member.createdByUser.firstName).to.equal('backer');
-      expect(collectiveData.members[2].member.slug).to.not.be.null;
-      expect(collectiveData.members[2].member.createdByUser.firstName).to.be.null;
-      expect(collectiveData.members[2].member.createdByUser.email).to.be.null;
-      expect(collectiveData.members[3].member.createdByUser.firstName).to.equal('host');
+      expect(collectiveData.members.length).to.equal(4);
+
+      const adminMember = collectiveData.members.find(m => m.member.id === adminUser.CollectiveId);
+      const backerMember = collectiveData.members.find(m => m.member.id === backerUser.CollectiveId);
+      const incognitoMember = collectiveData.members.find(m => m.member.id === incognitoCollective.id);
+      const hostMember = collectiveData.members.find(m => m.member.id === hostCollective.id);
+      expect(adminMember.member.createdByUser.firstName).to.equal('admin');
+      expect(backerMember.member.createdByUser.firstName).to.equal('backer');
+      expect(incognitoMember.member.slug).to.not.be.null;
+      expect(incognitoMember.member.createdByUser.firstName).to.be.null;
+      expect(incognitoMember.member.createdByUser.email).to.be.null;
+      expect(hostMember.member.createdByUser.firstName).to.equal('host');
```

Comment:
Had to update these tests because members return order is now fixed. Used something more robust.